src/rust/ide/view/graph-editor/src/component/node/output/port.rs

```diff
@@ -434,8 +456,33 @@ impl Model {
             eval opacity.value ((t) shape.set_opacity(*t));
 
 
+            // === Label Opacity ===
+
+            label_visibility        <- frp.on_hover.and(&frp.set_label_visibility);
+            label_opacity.target    <+ label_visibility.on_true().constant(PORT_OPACITY_HOVERED);
+            label_opacity.target    <+ label_visibility.on_false().constant(0.0);
+            label_color             <- all_with(&color.value,&label_opacity.value,
+                |color,&opacity| color.opaque.with_alpha(opacity).into());
+            label.set_color_all     <+ label_color;
+            label.set_default_color <+ label_color;
+
+
             // === Size ===
 
+            frp.source.size <+ frp.set_size;
```

Comment:
This is quite redundant. Can this be avoided?

Reply:
If its redundant, why not just delete it?

Reply:
I think I only tried to be consistent with other code and I find it slightly weird to use something called `set_size` in places where I only read the size.

Reply:
Felix, please do not mark such conversations as "resolved", as it was not resolved. You've replied, marked it as "resolved" and it was hidden. I'm reopenning it.

I still don't understand it. What do you mean by "This is quite redundant" then? I do not understand your last comment - could you elaborate, please? Let's not leave in code anything that is redundant. This needs to be fixed before merge.